website/content/docs/commands/license/index.mdx

```diff
@@ -26,14 +26,7 @@ performance_standby_count    9999
 start_time                   2021-05-17T00:00:00Z
 ```
 
-Query signed license string (only when using a deprecated stored license)
```

Comment:
Stored licenses aren't simply deprecated, at this point they are no longer supported (since `1.10`). See also: https://developer.hashicorp.com/vault/api-docs/system/license#stored